src/DataFixtures/ORM/LoadJecouteQuestionData.php

```diff
@@ -35,6 +36,9 @@ class LoadJecouteQuestionData extends Fixture
 
     public function load(ObjectManager $manager)
     {
+        AutoIncrementResetter::resetAutoIncrement($manager, 'jecoute_question');
```

Comment:
what about transform the second arg in array?

Reply:
we could. but would impact other fixtures. what about a refacto in another PR ?

Reply:
yes of course 👍